### PR TITLE
retriever condition improve for better response

### PR DESCRIPTION
**Pull request by JisanAR03**
The changes are very small, but we need them. Now our bot will respond more effectively. I was researching the chain and the vector retrieval part for a few hours. For now, they are fine to use.

@DonnieBLT , can you please this PR for merging

### CODE REVIEW

File: website/bot.py [hunk 1]
```diff
@@ -162,9 +162,9 @@ def conversation_chain(vector_store):
         )
     )
     llm = ChatOpenAI(model_name="gpt-3.5-turbo-0125", temperature=0.5)
-    retriever = vector_store.as_retriever(search_type="similarity", search_kwargs={"k": 3})
+    retriever = vector_store.as_retriever(search_type="similarity", search_kwargs={"k": 5})
     memory = ConversationSummaryMemory(
-        llm=llm, return_messages=True, memory_key="chat_history", max_token_limit=1000
+        llm=llm, return_messages=True, memory_key="chat_history", max_token_limit=1500
```
Comment (arkid15r) [inline website/bot.py:167]: Try using constants instead of magic numbers

Comment (JisanAR03) [reply website/bot.py:167]: ok sure

File: website/bot.py [hunk 2]
```diff
@@ -147,6 +147,9 @@ def load_vector_store():
 
 
 def conversation_chain(vector_store):
+    # not want to use magic numbers
+    search_kwargs = 5
```
Comment (arkid15r) [inline website/bot.py:151]: It's not clear from the name what's the meaning of this value. Could you make it more descriptive?

Comment (JisanAR03) [reply website/bot.py]: @arkid15r  update done sir

**Review by arkid15r — COMMENTED** [review]
It's still hard to tell what's `k`. Is it a limit for search results number?

LGTM

**Comment by JisanAR03** [comment]
> It's still hard to tell what's `k`. Is it a limit for search results number?
> LGTM

it's actually the retrieval result amount  from vector database based on the user question

**Review by DonnieBLT — APPROVED** [review]
looks straightforward so approving

**Pull request merged**